h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstNaRowCnt.java

```diff
@@ -0,0 +1,53 @@
+package water.rapids.ast.prims.reducers;
+
+import water.fvec.Frame;
+import water.fvec.Vec;
+import water.rapids.Env;
+import water.rapids.ast.AstPrimitive;
+import water.rapids.ast.AstRoot;
+import water.rapids.vals.ValNums;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This method counts the number of rows that contain NAs in a frame.  It does not count
+ * each occurrence of the NA but rather the number of rows that contain at least one NA.
+ */
+public class AstNaRowCnt extends AstPrimitive {
+  @Override
+  public String[] args() {
+    return new String[]{"ary"};
```

Comment:
Not sure.  Looked at the ast/prims/reducers.  Operations on frames sometimes return "ary", sometimes return "frame".  Have no idea what all this is all about.  Will replace "ary" with "frame" in our case since we do play with frames and this operation is only meaningful for frames.